src/heads-up/components/property-data.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <span>{{ tooltipMessage }}</span>
+</template>
+<script>
+
+function isDefined(tag) {
+  const isImg = tag === 'og:image'
+  return `There is no ${tag} defined. You can create the ${tag} in the <head> like <meta property="${tag}" content="${isImg?'your source':'your content'}."> `
```

Comment:
I would probably put this: `${isImg?'your source':'your content'}` in a variable in this function and write it like `const metaTagContent = isImg ? 'your source' : 'your content`